### DISCUSSION TITLE
Autofit of generated tables in docx when converting from MD, using reference.docx and yaml file

### DISCUSSION THREAD

**Discussion by zero0n3** [jgm/pandoc#10425] [General]
Hello

I insert the following code to generate a table during conversion from md to docx
```
| Trans.   | Code         | Description                                              |
| :------- | :----------- | :------------------------------------------------------- |
| AA00AA02 | CBBO00/01    | Daily journal monthly movements corr. foo bar            |
| AA00AA03 | CGBO*        | Journal FOO BAR                                          |
| AA00AA04 | DOCPA3-CONTF | foo bar signed allo box office  (migration from AA00A01) |
```

The result of the conversion is as follows:

![immagine](https://github.com/user-attachments/assets/2019ffd5-2e40-445d-8832-6216c08be17d)

As you can see, the columns do not have a correct spacing, especially the first column. So on Word I then use the AutoFit command and then the AutoFitWindow.

Working on the reference.docx file or on another file (for example the yaml for the default), would it be possible, in your opinion, to set the use of these two commands, during the creation of the final converted docx?

Or are there other solutions to have the autofit of the tables during the conversion from md to docx? (for example inserting a macro in the reference?)

Thanks

**Comment by jgm** [comment]
Please see [the documentation](https://pandoc.org/MANUAL.html#extension-pipe_tables) for pipe tables.  You just need to adjust the relative widths of the separator lines, increasing the width of the first and second ones and decreasing the width of the third.  The separator lines need not align with the content.

**Reply by zero0n3** [reply]
Thanks @jgm this is a perfect solution 😄
